### PR TITLE
Update card-sort.md

### PR DESCRIPTION
**Pull request by VitaGit**

### CODE REVIEW

File: research-activities/card-sort.md [hunk 1]
```diff
@@ -1,20 +1,42 @@
...
+You can also think of these as two parts of the design process:
+Use open sorts to understand what goes together and what it should be called. 
+Use closed sorts to test and validate what you already have with more people and other types of users. 
 
-TODO: we need some photographs and example outputs.
+Hybrid sessions are also useful. You provide categories but let people create new ones, and remove and rename categories if they feel it doesn’t fit into any of the existing ones.
+
+## Instructions (open sort)
+
+1. Prepare category cards for each participant. *Write categories on cards for a closed sort.
```
Comment (tahb) [inline research-activities/card-sort.md]: Where do the categories come from? How would I know what to include or what granularity?

File: research-activities/card-sort.md [hunk 2]
```diff
@@ -1,20 +1,42 @@
...
+2. Prepare some content cards for each category (a set for each participant)
+3. Ask participants to group the cards in the way that makes sense to them and give the groups names. *For a closed sort participants move content cards into categories you’ve prepared.
+4. Discuss why they grouped the cards the way they did
```
Comment (tahb) [inline research-activities/card-sort.md]: Presumably the note taker whirls into action at this point. Any tips for them at this point, what are they trying to record? Perhaps a link to your note taking piece would set this up nicely?

File: research-activities/card-sort.md [hunk 3]
```diff
@@ -1,20 +1,42 @@
...
+5. If there too many categories and time allows, you can try to get participants to reach a consensus
+6. Repeat
+
```
Comment (tahb) [inline research-activities/card-sort.md]: Do we need instructions for a closed sort, does it differ?

File: research-activities/card-sort.md (hunk 2 above)
Comment (VitaGit) [inline research-activities/card-sort.md]: The note taker will likely take most notes here, but often there are questions and comments made while sorting is in progress. Those are equally and often more important to capture.

**Comment by VitaGit** [comment]
@tahb I've made changes based on your comments. Please have another look

File: research-activities/card-sort.md (hunk 1 above)
Comment (VitaGit) [inline research-activities/card-sort.md]: I've included extra information where the categories might come from such as previous rounds of sorting, design sprints. I've also included a 'granularity' sentence in 'things to remember'

**Pull request merged**